packages/core/integration-tests/test/glob.js

```diff
@@ -203,4 +203,39 @@ describe('glob', function () {
       ],
     });
   });
+
+  it('should require a glob of files from a package', async function () {
+    let b = await bundle(
+      path.join(__dirname, '/integration/glob-package/index.js'),
+    );
+    await assertBundles(b, [
+      {
+        name: 'index.js',
+        assets: ['*.js', '*.js', 'a.js', 'b.js', 'x.js', 'y.js', 'index.js'],
+      },
+    ]);
```

Comment:
Probably should `run` the resulting bundle here as in the other tests to ensure it actually works